core/templates/services/page-title.service.ts

```diff
@@ -24,8 +24,10 @@ import { downgradeInjectable } from '@angular/upgrade/static';
   providedIn: 'root'
 })
 export class PageTitleService {
-  pageTitleForMobile: string = null;
-  pageSubtitleForMobile: string = null;
+  // The following properties are 'null' when no title needs to
+  // displayed in the mobile view.
+  pageTitleForMobile: string | null = null;
```

Comment:
Can we do non-null assertion here ?

Reply:
Didn't you also migrate this file in a different PR?

Reply:
Ok, same comments as in the other PR also apply here.